qutebrowser/config/configdata.yml

```
@@ -1091,6 +1091,20 @@ hints.border:
   type: String
   desc: CSS border value for hints.
 
+hints.padding:
+  default:
+    top: 0
+    bottom: 0
+    left: 3
+    right: 3
+  type: Padding
+  desc: Padding (in pixels) for hints.
+
+hints.radius:
+  default: 0
+  type: Int
```

```suggestion
  type:
    name: Int
    minval: 0
```
[Other radii prevent negative values](https://github.com/qutebrowser/qutebrowser/blob/df45e7401ee76a5243a260179a32e0a11c330a9b/qutebrowser/config/configdata.yml#L1361L1363).